script/scaffold/templates/config_flow/integration/__init__.py

```diff
@@ -8,7 +9,7 @@
 
 # TODO List the platforms that you want to support.
 # For your initial PR, limit it to 1 platform.
-PLATFORMS = ["light"]
+PLATFORMS: List[Text] = ["light"]
```

Comment:
```suggestion
PLATFORMS: list[str] = ["light"]
```

Reply:
Thanks! I'm a bit new to Python and didn't notice it can be done better 😄